setup-pyenv.sh

```diff
@@ -13,6 +13,8 @@
 #     Release tag of pyenv to download [default: clone from master]
 # - PYENV_CACHE_PATH
 #     Directory where full Python builds are cached (i.e., for Travis)
+# - VIRTUALENV_EXTRA_ARGS
+#     Extra options to be handed when we setup Virtualenv
```

Comment:
Can you make this `Extra arguments to be used when creating the virtualenv`?

Also, this needs to be added to the README.

Reply:
Done